lxd/auth/oidc/oidc.go

```diff
@@ -302,13 +305,18 @@ func getAccessTokenVerifier(issuer string) (op.AccessTokenVerifier, error) {
 
 // NewVerifier returns a Verifier.
 func NewVerifier(issuer string, clientid string, audience string) (*Verifier, error) {
-	cookieKey, err := uuid.New().MarshalBinary()
-	if err != nil {
-		return nil, fmt.Errorf("Failed to create UUID: %w", err)
+	cookieHashKey := securecookie.GenerateRandomKey(32)
+	cookieBlockKey := securecookie.GenerateRandomKey(32)
+	secureCookie := securecookie.New(cookieHashKey, cookieBlockKey)
+
+	verifier := &Verifier{
+		issuer:         issuer,
+		clientID:       clientid,
+		audience:       audience,
+		cookieHashKey:  securecookie.GenerateRandomKey(32),
```

Comment:
No I meant to use the same one. It doesn't cause an error because the cookie hash and block keys are used as part of the login flow (/oidc/login, /oidc/callback), whereas the secure cookie field is used for encrypting the user id and refresh tokens when authenticating afterwards.

I'll change it now to use the same one in any case.